site/public/js/pdf/PDFInitToolbar.js

```diff
@@ -187,7 +189,7 @@ function renderPDFToolbar() {
         document.getElementById("size_selector").addEventListener('click', sizeMenuToggle);
         document.addEventListener('colorchange', changeColor);
         let init_color = localStorage.getItem('main_color');
-        setColor(init_color);
+        setColor(init_color ? init_color : '#000000');
```

Comment:
if `init_color` is either a string or undefined, can do the following shorthand in JS:
```suggestion
        setColor(init_color || '#000000');
```